e2e-tests/IIoTPlatform-E2E-Tests/TestConstants.cs

```diff
@@ -139,6 +139,31 @@ internal static class EnvironmentVariablesNames {
             /// IoT Hub connection string
             /// </summary>
             public const string PCS_IOTHUB_CONNSTRING = "PCS_IOTHUB_CONNSTRING";
+
+            /// <summary>
+            /// Docker server
+            /// </summary>
+            public const string PCS_DOCKER_SERVER = "PCS_DOCKER_SERVER";
+
+            /// <summary>
+            /// Docker user name
+            /// </summary>
+            public const string PCS_DOCKER_USER = "PCS_DOCKER_USER";
+
+            /// <summary>
+            /// Docker password 
+            /// </summary>
+            public const string PCS_DOCKER_PASSWORD = "PCS_DOCKER_PASSWORD";
+
```

Comment:
The name is misleading, we should use CONTAINER_REGISTRY instead of DOCKER

Reply:
I know. I used the same name that was used in IIoT.
Changed

Reply:
@dacolgit, @koepalex, let's keep the `DOCKER` in the names instead of `CONTAINER_REGISTRY ` as we already have those configuration parameters for the platform.

Reply:
@karok2m @dacolgit - I would not reuse them because, their are not persisted in KeyVault and their are misleading. The new name is more clear/understandable even if someone don't know it was called `DOCKER` in configuration parameters of the platform

Reply:
OK, let's go with `CONTAINER_REGISTRY` then.